# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1465928889 14400
#      Tue Jun 14 14:28:09 2016 -0400
# Node ID d25e9f85ffb491adb258d69a1f9d11b2fe75ddd2
# Parent  7c053ad1c7ba03b892d1dd71a1176d835800c045
distribute liboctave/wrappers header files

* liboctave/wrappers/module.mk (WRAPPERS_SRC): Include
$(NOINSTALL_WRAPPERS_INC) in the list.

diff --git a/liboctave/wrappers/module.mk b/liboctave/wrappers/module.mk
--- a/liboctave/wrappers/module.mk
+++ b/liboctave/wrappers/module.mk
@@ -29,17 +29,18 @@ WRAPPERS_SRC = \
   liboctave/wrappers/nproc-wrapper.c \
   liboctave/wrappers/putenv-wrapper.c \
   liboctave/wrappers/set-program-name-wrapper.c \
   liboctave/wrappers/strftime-wrapper.c \
   liboctave/wrappers/strptime-wrapper.c \
   liboctave/wrappers/uname-wrapper.c \
   liboctave/wrappers/unsetenv-wrapper.c \
   liboctave/wrappers/vasprintf-wrapper.c \
-  liboctave/wrappers/wait-for-input.c
+  liboctave/wrappers/wait-for-input.c \
+  $(NOINSTALL_WRAPPERS_INC)
 
 noinst_LTLIBRARIES += liboctave/wrappers/libwrappers.la
 
 liboctave_wrappers_libwrappers_la_SOURCES = $(WRAPPERS_SRC)
 
 liboctave_wrappers_libwrappers_la_CPPFLAGS = \
   -Ilibgnu -I$(srcdir)/libgnu
 
